main.go

```diff
@@ -305,15 +301,43 @@ func mainErr(args []string) error {
 			fmt.Fprintf(os.Stderr, "usage: garble version\n")
 			return errJustExit(2)
 		}
-		// don't overwrite the version if it was set by -ldflags=-X
-		if info, ok := debug.ReadBuildInfo(); ok && version == "(devel)" {
-			mod := &info.Main
-			if mod.Replace != nil {
```

Comment:
note to self: we still need this, because it's indeed possible for the module containing `package main` to be replaced.

Reply:
Done.